torch/_inductor/fx_passes/serialized_patterns/_sfdp_pattern_1.py

```diff
@@ -100,7 +100,7 @@
 expand_default_3 = CallFunction(aten.expand.default, KeywordArg('value'), Ignored())
 view_default_4 = CallFunction(aten.view.default, expand_default_3, Ignored())
 bmm_default_1 = CallFunction(aten.bmm.default, view_default_3, view_default_4)
-_sfdp_pattern_1_inference = CallFunction(aten.view.default, bmm_default_1, Ignored())
+_sfdp_pattern_1_inference = CallFunction(aten.view.default, bmm_default_1, Ignored(), _users=0)
```

Comment:
Is there a reason to add `_user=0` here ?

Reply:
As mentioned in the summary - the patterns weren't being serialized properly before and the test didn't catch it. This is the output of the generator now that it's correctly serializing the pattern.